stan/math/prim/prob/pareto_type_2_lccdf.hpp

```diff
@@ -18,60 +20,83 @@ return_type_t<T_y, T_loc, T_scale, T_shape> pareto_type_2_lccdf(
     const T_y& y, const T_loc& mu, const T_scale& lambda,
     const T_shape& alpha) {
   using T_partials_return = partials_return_t<T_y, T_loc, T_scale, T_shape>;
-  using std::log;
+  using T_y_ref = ref_type_if_t<!is_constant<T_y>::value, T_y>;
```

Comment:
operands_and_partials only does anything with autodiff arguments.

Reply:
So if `T_y` is not autodiff, what does `T_y_ref` end up being?

Reply:
It ends up being `const T_y&`.

Reply:
But if it's an expression for doubles wouldn't you still want to evaluate it to a simple expression (via to_ref)?

I'm guessing the answer is no, but I don't see why.

Reply:
I don't need to. It will be evaluated on line 51, as a part of larger expression, which will also transpose it if it is a row vector and make it into array, so I can do elementwise operations.

Reply:
Right. Thanks for the patience